src/_gettsim/interface.py

```diff
@@ -540,10 +540,10 @@ def _round_and_partial_parameters_to_functions(functions, params, rounding):
         if partial_params:
             partial_func = functools.partial(function, **partial_params)
 
-            # Make sure that rounding parameter attribute is transferred to partial
+            # Make sure any GETTSIM metadata is transferred to partial
             # function. Otherwise, this information would get lost.
-            if hasattr(function, "__rounding_params_key__"):
-                partial_func.__rounding_params_key__ = function.__rounding_params_key__
+            if hasattr(function, "__gettsim__"):
+                partial_func.__gettsim__ = function.__gettsim__
 
             processed_functions[name] = partial_func
         else:
```

Comment:
`functools.wraps` could _probably_ be used here, however so we don't need the `hasattr` and manual check, however.

Some test code:
```py
from functools import wraps, partial

def f(x):
    return x

f.test = "test"

g = wraps(f)(partial(f, 1))

print(g.test) # Output: test
print(g()) # Output: 1
```

Reply:
I don't find `partial_func = functools.wraps(function)(functools.partial(function, **partial_params))` particularly readable, though.

Reply:
Definitely not! 

*(I am afraid you'll have to get used to me throwing around some wild ideas without fully understanding what's going on. Occasionally I do make a useful hit...)*